sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerRunner.java

```diff
@@ -101,6 +102,10 @@ public void run() {
         CuratorLocker locker = new CuratorLocker(schedulerBuilder.getServiceSpec());
         locker.lock();
         try {
+            SchedulerConfig schedulerConfig = SchedulerConfig.fromEnv();
```

Comment:
It "shouldn't happen", but it could make sense to have a try/catch around this section for the case where STATSD_UDP_HOST/_PORT aren't set